core/MusicControl/src/main/java/org/jjazz/musiccontrol/api/playbacksession/UpdateProviderSongSession.java

```diff
@@ -414,19 +411,22 @@ public void propertyChange(PropertyChangeEvent e)
                         doUpdate = true;
                     }
 
-
                     //
                     // PlaybackSettings source events
                     //       
                     case PlaybackSettings.PROP_CLICK_PITCH_HIGH, PlaybackSettings.PROP_CLICK_PITCH_LOW, PlaybackSettings.PROP_CLICK_PREFERRED_CHANNEL, PlaybackSettings.PROP_CLICK_VELOCITY_HIGH, PlaybackSettings.PROP_CLICK_VELOCITY_LOW, PlaybackSettings.PROP_CLICK_PRECOUNT_MODE, PlaybackSettings.PROP_CLICK_PRECOUNT_ENABLED ->
                     {
                         dirty = true;
                     }
-                    case PlaybackSettings.PROP_PLAYBACK_KEY_TRANSPOSITION, PlaybackSettings.PROP_PLAYBACK_CLICK_ENABLED ->
+                    case PlaybackSettings.PROP_PLAYBACK_CLICK_ENABLED ->
                     {
                         doUpdate = true;
                     }
-
+                    case PlaybackSettings.PROP_DISPLAY_TRANSPOSITION ->
```

Comment:
yes, before it had an impact on playback. Now I think it's better to remove the case.